snuba/datasets/configuration/group_attributes/storages/group_attributes.yaml

```diff
@@ -0,0 +1,55 @@
+version: v1
+kind: writable_storage
+name: group_attributes
+
+storage:
+  key: group_attributes
+  set_key: group_attributes
+
+readiness_state: limited
+
+schema:
+  columns:
+    [
+      { name: project_id, type: UInt, args: { size: 64 } },
+      { name: group_id, type: UInt, args: { size: 64 } },
+
+      { name: group_status, type: UInt, args: { size: 8 } },
+      { name: group_substatus, type: UInt, args: { size: 8, schema_modifiers: [ nullable ] } },
+      { name: group_first_seen, type: DateTime },
+      { name: group_num_comments, type: UInt, args: { size: 64 } },
+
+      { name: assignee_user_id, type: UInt, args: { size: 64, schema_modifiers: [ nullable ] } },
+      { name: assignee_team_id, type: UInt, args: { size: 64, schema_modifiers: [ nullable ] } },
+
+      { name: owner_suspect_commit_user_id, type: UInt, args: { size: 64, schema_modifiers: [ nullable ] } },
+      { name: owner_ownership_rule_user_id, type: UInt, args: { size: 64, schema_modifiers: [ nullable ] } },
+      { name: owner_ownership_rule_team_id, type: UInt, args: { size: 64, schema_modifiers: [ nullable ] } },
+      { name: owner_codeowners_user_id, type: UInt, args: { size: 64, schema_modifiers: [ nullable ] } },
+      { name: owner_codeowners_team_id, type: UInt, args: { size: 64, schema_modifiers: [ nullable ] } },
+
+      { name: deleted, type: UInt, args: { size: 8 } },
+      { name: message_timestamp, type: DateTime },
+      { name: partition, type: UInt, args: { size: 16 } },
+      { name: offset, type: UInt, args: { size: 64 } },
+    ]
+  local_table_name: group_attributes_local
+  dist_table_name: group_attributes_dist
+
+allocation_policies:
+  - name: PassthroughPolicy
+    args:
+      required_tenant_types:
+        - blank
```

Comment:
In full prod version you should use the BytesScannedWindowAllocationPolicy to throttle queries by org above a certain threshold of bytes scanned. We can discuss further what this means if you'd like.

Reply:
Yup, that makes sense to me. Will create a follow-up PR with more robust settings for prod when we're closing to launching this dataset.